README.md

```diff
@@ -104,7 +107,7 @@ You're a good person and decide to validate your HTML. You copy and paste the co
 The validator throws errors at you! Why? Assuming you haven't made any mistakes in your code, how could you go about accurately validating your HTML?
 
 ```
-Your answer...
+You also have ruby syntax in the html code . you could use a form validator for ruby.
```

Comment:
You're half-right. You would need to "View Source" on the page or "Inspect Element" on the page and copy the HTML.